charts/trusted-certificate-issuer/templates/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  sopin: V0lwbUJCybc2Oc6M06Vz
```

Comment:
I think we should make token pins configurable.

Reply:
done

Reply:
> done

Thanks.  We should also list these new params in the ReadME.

Reply:
added